test/integration/cart_integration_test.rb

```diff
@@ -3,6 +3,14 @@
 class CartIntegrationTest < ActionDispatch::IntegrationTest
   include Capybara::DSL
 
+  test 'a cart starts empty' do
+    visit '/'
+    click_link_or_button('Cart')
+    within('#cart_items') do
+      assert page.has_content?('Your Cart Is Empty')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.